ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include <fstream>
 
+namespace gcs = ::google::cloud::storage;
```

Comment:
let's put using namespace gsc = ::google::cloud::storage here, so we can remove the "using namespace" below.

Reply:
addressed here: 0adccbf3c97d00eb573380c9a26490d45ef417dc